### PR TITLE
added stable diffusion api call

### LINKED ISSUE #36: Update API call to New Stable Diffusion API and take in text param
As a user, I want to be able to fetch an image from a [Dezgo API ](https://rapidapi.com/dezgo/api/dezgo/details)using inputted text and parameters.

### PR DESCRIPTION
**Pull request by anamika8**
Added the ability to call the Stable Diffusion API with a parameter.
Integrated the call and subsequent image rendering via React to ensure that the call is happening successfully. For validating the changes, you can pull this branch to your local and run npm start

The API call and React rendering are fault-tolerant such that any API error or rendering error will result in displaying the previous image set by Stuart in his PR

Below is what can be seen on the landing page now

Screen Shot 2023-02-21 at 1 39 44 PM

### CODE REVIEW

File: artscii/src/services/stableDiffusionService.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    const imageBase64 = await callStableDiffusionWithPrompt(encodedParams);
+    if (imageBase64 === undefined) {
+        console.log(`No response from Dezgo API - hence displaying default image url`);
+        return DEFAULT_IMAGE_TO_SHOW;
+    }
+    var image = `data:image/png;base64, ${imageBase64}`;
+    return image;
+}
+
+// function to call the Giphy API
```
Comment (srimel) [inline artscii/src/services/stableDiffusionService.js]: NIT: Outdated comment, recommend to change or delete

Comment (anamika8) [reply artscii/src/services/stableDiffusionService.js]: Good catch. Thanks for the suggestion!

File: artscii/src/components/main/Main.js [hunk 2]
```diff
@@ -1,53 +1,76 @@
...
+            .then(imageUrl => {
+                console.log(`Image URL received in UI - ${imageUrl}`)
+                setImageUrl(imageUrl)
+            })
+            .catch(err => {
+                console.log("error encountered = " + err);
+            });
+    }
+
+    return ( <
```
Comment (unfinishedideas) [inline artscii/src/components/main/Main.js:41]: Something happened to the formatting here :/

Comment (unfinishedideas) [inline artscii/src/components/main/Main.js]: Something messed up the formatting here :/

Comment (anamika8) [reply artscii/src/components/main/Main.js]: Fix the formatting in main.js file

**Review by elizasohn — APPROVED** [review]

**Pull request merged**